core/src/main/scala/com/databricks/labs/remorph/coverage/CoverageTest.scala

```diff
@@ -67,18 +51,13 @@ object Main {
           commit_hash = commitHash,
           version = "latest",
           timestamp = now.toString,
-          source_dialect = sourceDialect,
-          target_dialect = targetDialect,
+          source_dialect = dialect,
+          target_dialect = "databricks",
           file = os.Path(test.inputFile).relativeTo(sourceDir).toString)
         val report = runner.runQuery(exampleQuery)
         val reportEntryJson = ReportEntry(header, report).asJson
         os.write.append(outputFilePath, ujson.write(reportEntryJson, indent = -1) + "\n")
       }
     }
-    // scalastyle:off
-    println(s"Successfully produced coverage report in $outputFilePath")
-    // scalastyle:on
```

Comment:
can you also print the coverage %? either in this PR or the next one. then https://github.com/databrickslabs/remorph/blob/main/src/databricks/labs/remorph/coverage/local_report.py and https://github.com/databrickslabs/remorph/blob/main/src/databricks/labs/remorph/coverage/commons.py#L62-L78 could be removed

Reply:
I will reintroduce that `println` (in an improved version with %) in the next PR